### PR TITLE
Orchestration templates validation test cases

### PR DESCRIPTION
**Pull request by niyazRedhat**
{{pytest: -v -k 'test_name_required_error_validation or test_all_fields_required_error_validation or test_new_template_required_error_validation'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3f8faf46981a4ec4aeb083e0f7a245e33f12e631:

`cfme/tests/services/test_orchestration_template.py`:
- :bangbang: Line 115:101: **E501** *line too long (110 > 100 characters)*

Please, rectify these issues :smirk: .
*CFME QE Bot*

File: cfme/tests/services/test_orchestration_template.py [hunk 1]
```diff
@@ -39,6 +40,9 @@
 """
 
 
+METHOD_TORSO_empty = ""
```
Comment (mfalesni) [inline cfme/tests/services/test_orchestration_template.py]: I think you don't have to use a constant for an empty string :)

Comment (niyazRedhat) [inline cfme/tests/services/test_orchestration_template.py]: @milan What value i have  to use?

Comment (mfalesni) [inline cfme/tests/services/test_orchestration_template.py]: @niyazRedhat hehe, you mentioned `@milan` and not me. Well, for an empty string, you don't have to have some specific constant somewhere, just use an empty string where you want to use it.

btw. constants' names must be all uppercase.

**Comment by dajoRH** [comment]
Lint report for commit 7939ec204bb42e6243375e19380382b9252cf2a1:

`cfme/tests/services/test_orchestration_template.py`:
- :bangbang: Line 115:101: **E501** *line too long (110 > 100 characters)*

Please, rectify these issues :smirk: .
*CFME QE Bot*

File: cfme/tests/services/test_orchestration_template.py [hunk 2]
```diff
@@ -80,3 +84,33 @@ def test_copy_template(provisioning):
...
+        template.create(METHOD_TORSO)
+
+
+def test_new_template_required_error_validation(provisioning):
+    template_type = provisioning['stack_provisioning']['template_type']
+    template = OrchestrationTemplate(template_type=template_type,
+                                     template_name=fauxfactory.gen_alphanumeric(),
+                                     description="my template")
+
+    with error.expected("Error during Orchestration Template creation: new template content cannot be empty"):
```
Comment (mfalesni) [inline cfme/tests/services/test_orchestration_template.py]:
```python
with error.expected(
        'Error during Orchestration Template creation: new template content cannot be empty'):
    template.create(...)
```

**Comment by dajoRH** [comment]
Lint report for commit 1a03d05fdb27ab8c75f5ffd0812f755bf7dd8a95:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7e3ba94d816d8105fc9921d7fdddb7768b1e6af8:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5a6100693db8b530972b37369604ef430f772d66:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**